### PR TITLE
Feat/styling setup

### PR DESCRIPTION
**Pull request by Hornebom**
- refactors color values and typography classes 
- adds logo image files
- adds favicons
- add fonts

### CODE REVIEW

File: src/assets/styles/base/_layout.scss [hunk 1]
```diff
@@ -31,8 +31,16 @@ body {
 }
 
 .container {
-  width: 90%;
+  width: 100%;
```
Comment (bob-voorhoede) [inline src/assets/styles/base/_layout.scss:34]: Oké, we're in the early stages of design, but shall we think about mobile first? A 100% container will mean it stretches the full width, until it reaches max width. That is not the purpose of the container; it is to keep stuff "off" the edges and center the content.

File: src/assets/styles/base/_typography.scss [hunk 2]
```diff
@@ -8,27 +8,45 @@ h6 {
...
+// sizes
+.h1 {
+  font-size: 5rem;
+}
+
+.h2 {
+  font-size: 3rem;
+}
+
+.h3 {
```
Comment (bob-voorhoede) [inline src/assets/styles/base/_typography.scss:20]: +1 for using classes 🥳

File: src/static/browserconfig.xml [hunk 3]
```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (bob-voorhoede) [inline src/static/browserconfig.xml:1]: I see this file added but not referenced in site meta - doesn't it need a meta tag with `msapplication-config`?

**Review by bob-voorhoede — APPROVED** [review]
Some tiny remarks, pls merge when addressed ✅ 🌵

File: src/static/browserconfig.xml (hunk 3 above)
Comment (bob-voorhoede) [reply src/static/browserconfig.xml:1]: This file is also referenced (commented out) in `config/build-scripts/revision.js` line 49 - do we need to do something with that?

File: src/assets/styles/base/_layout.scss (hunk 1 above)
Comment (Hornebom) [reply src/assets/styles/base/_layout.scss:34]: I will use the .container class to handle max-width and centering. Another class will handle paddings

File: src/assets/styles/base/_typography.scss (hunk 2 above)
Comment (Hornebom) [reply src/assets/styles/base/_typography.scss:20]: 🎉

File: src/static/browserconfig.xml (hunk 3 above)
Comment (Hornebom) [reply src/static/browserconfig.xml:1]: Looks like a meta tag is needed
https://docs.microsoft.com/en-gb/previous-versions/windows/internet-explorer/ie-developer/platform-apis/dn320426(v=vs.85)

Comment (Hornebom) [reply src/static/browserconfig.xml:1]:
```html
<meta name="msapplication-config" content="IEconfig.xml" />
```

Comment (Hornebom) [reply src/static/browserconfig.xml:1]: will also enable the revision part

**Pull request merged**